tests/common/devices.py

```diff
@@ -1070,10 +1070,14 @@ def get_namespace_from_asic_id(self, asic_id):
             return DEFAULT_NAMESPACE
         return "{}{}".format(NAMESPACE_PREFIX, asic_id)
 
-    def get_extended_minigraph_facts(self, tbinfo):
+    def get_extended_minigraph_facts(self, tbinfo=None):
```

Comment:
When do we not need the tbinfo?

Reply:
The comments is in the code below. Some callers they don't use the port map on the ptf, then they don't need to provide tbinfo for the look up.

Reply:
@neethajohn thanks for the advice. Made further change to make tbinfo mandatory.